recipe/meta.yaml

```diff
@@ -25,51 +25,31 @@ requirements:
     - wheel
   run:
     - python
-    - arrow <3.0,>=1.2.0
-    - backoff <4.0,>=2.2.1
-    - beautifulsoup4 <6.0,>=4.8.0
-    - click <10.0
-    - croniter <1.5.0,>=1.3.0
-    - dateutils <2.0
-    - deepdiff <8.0,>=5.7.0
-    - fastapi <2.0,>=0.92.0
-    - fsspec <2025.0,>=2022.5.0
-    - inquirer <5.0,>=2.10.0
-    - jinja2 <5.0
-    - lightning-cloud >=0.5.38
-    - lightning-utilities <2.0,>=0.7.0
-    - numpy <3.0,>=1.17.2
-    - packaging
-    - psutil <7.0
-    - pydantic <2.2.0,>=1.7.4
-    - python-multipart <2.0,>=0.0.5
-    - pytorch <4.0,>=1.11.0
+    - pyyaml >=5.4,<8.0
+    ## fsspec[http] ##
+    - fsspec >=2022.5.0,<2026.0
+    - aiohttp !=4.0.0a0,!=4.0.0a1
+    - requests
+    ## fsspec[http] ##
+    - lightning-utilities >=0.10.0,<2.0
+    - numpy >=1.17.2,<3.0
+    - packaging >=20.0,<25.0
+    - pytorch >=2.0.0,<4.0
+    - torchmetrics >=0.7.0,<3.0
+    - tqdm >=4.57.0,<6.0
+    - typing-extensions >=4.4.0,<6.0
     - pytorch-lightning
-    - pyyaml <8.0
-    - requests <4.0
-    - rich <15.0,>=12.3.0
-    - starlette
-    - starsessions <2.0,>=1.2.1
-    - torchmetrics <3.0,>=0.7.0
-    - tqdm <6.0,>=4.57.0
-    - traitlets <7.0,>=5.3.0
-    - typing-extensions <6.0,>=4.0.0
-    - urllib3 <4.0
-    - uvicorn <2.0
-    - websocket-client <3.0
-    - websockets <13.0
+    - click
```

Comment:
Do we need `click`, here?  It's in a `fabric` [test requirements](https://github.com/Lightning-AI/pytorch-lightning/blob/master/requirements/fabric/test.txt#L8).

Reply:
`click` is needed to builded the entry points, which I do use for testing.

Reply:
I think it should be safe to move `click` to `test.requires`. I'm concerned we may be missing a few dependencies needed for runtime, but unfortunately I cannot do a deep dive on this today.